python/setup.py

```diff
@@ -11,6 +11,7 @@
     py_modules=[],
     url='https://github.com/dropbox/mypy-protobuf',
     download_url='https://github.com/dropbox/mypy-protobuf/archive/v1.13.tar.gz',
+    install_requires=['protobuf>=3.7.1'],
```

Comment:
Is this the minimal version allowed? In my organization we're still on protobuf 3.6 in some places.

Reply:
ah - I'm not sure. I know that 3.5 does not work, but did not try 3.6. This is the minimal version which is tested. (see https://github.com/dropbox/mypy-protobuf/blob/master/requirements.txt)

Reply:
3.5 yielded an error at import time, so I couldn't test it
```
E   TypeError: __new__() got an unexpected keyword argument 'serialized_options'
```

3.6 has some small functional differences, but is probably mostly compatible in most cases
```
>       s.ClearField(u"a_string")
E       TypeError: field name must be a string
```

Seems like we may just want to require the latest tested version here, though your organization may be able to get away with a lower version - as long as you don't rely on the minor divergences we know of (unicode vs str on ClearField)
Note that this is not protoc version, but rather the python-protobuf version. (which are numbered similarly and can be confusing)

Reply:
Is mypy-protobuf calling `ClearField` with a `unicode` as part of its operation?